wagtail/admin/tests/pages/test_move_page.py

```diff
@@ -241,3 +244,22 @@ def hook_func(request, page):
         self.assertEqual(
             Page.objects.get(id=self.test_page_a.id).get_parent().id, self.section_b.id
         )
+
+    def test_page_move_after_parent_page_types_changes_to_different_parent_model(self):
+        page = self.section_a.add_child(
+            instance=BusinessSubIndex(
+                title="Business Sub Index",
+                slug="business-sub-index",
+                live=True,
+                has_unpublished_changes=False,
+            )
+        )
+
+        response = self.client.get(
```

Comment:
**Suggestion**: Because the validity of this depends on the fixtures not changing, let's check it's a valid test, too. I'm not sure which of `.can_exist_at`, `.can_move_to`, and `.can_create_at` is the appropriate test, but:

```suggestion
        self.assertFalse(BusinessSubIndex.can_exist_at(self.section_a))
        
        response = self.client.get(
```

Reply:
great suggestion. it was `.can_exist_under()` that we wanted. Added a comment with a reference to the issue as well

Reply:
under/at :facepalm: Thanks!